rasa_sdk/executor.py

```diff
@@ -26,21 +26,21 @@ def __init__(self) -> None:
 
     def utter_message(
         self,
+        text: Optional[Text] = None,
         image: Optional[Text] = None,
         json_message: Dict[Text, Any] = None,
         template: Optional[Text] = None,
         attachment: Optional[Text] = None,
-        text: Optional[Text] = None,
         buttons: Optional[List[Dict[Text, Any]]] = None,
-        *elements: Dict[Text, Any],
+        elements: Optional[List[Dict[Text, Any]]] = None,
         **kwargs: Any,
     ) -> None:
         """"Send a text to the output channel"""
 
         message = {
             "text": text,
-            "buttons": buttons,
-            "elements": elements,
+            "buttons": buttons or [],
+            "elements": elements or [],
             "custom": json_message,
```

Comment:
```suggestion
            "custom": json_message or {},
```
Not sure

Reply:
Do we need this? I mean we actually don't even need it for the lists 👀

Reply:
will add it

Reply:
yeah i was wondering if we needed it for the lists as well haha